docs/supported-mutators.md

```diff
@@ -204,6 +206,8 @@ Due to differences in language syntax, method expressions are implemented differ
 | MaxToMin                        | `Max()`               | `Min()`             |
 | SumToCount                      | `Sum()`               | `Count()`           |
 | CountToSum                      | `Count()`             | `Sum()`             |
+| Append                          | `Append()`            | `Prepend()`         |
+| Prepend                         | `Prepend()`           | `Append()`          |
```

Comment:
```suggestion
| AppendToPrepend                          | `Append()`            | `Prepend()`         |
| PrependToAppend                         | `Prepend()`           | `Append()`          |
```

Alternatively 

```suggestion
| AppendNegation                          | `Append()`            | `Prepend()`         |
| PrependNegation                         | `Prepend()`           | `Append()`          |
```

However I'm not sure if that change sufficiently natural for usage of Negation. We also did not use Negate for Any and All so I think I prefer the To variant. @nicojs what do you think